hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/shell/TestBulkDeleteCommand.java

```diff
@@ -64,4 +78,52 @@ public void testArguments() throws IOException, URISyntaxException {
         Assertions.assertThat(bulkDeleteCommand.childArgs.get(0)).
                 describedAs("Children arguments must match").isEqualTo(arg2);
     }
+
+    @Test
+    public void testLocalFileDeletion() throws IOException {
+        String deletionDir = "toDelete";
+        String baseFileName = "file_";
+        Path baseDir = new Path(testRootDir, deletionDir);
+        List<String> listOfPaths = new ArrayList<>();
+        for(int i = 0; i < 100; i++) {
+            Path p = new Path(baseDir, baseFileName + i);
```

Comment:
there are utilities for create files and directories in ContractTestUtils.

Reply:
these are empty files, should I use touch from ContractTestUtils here?

Reply:
yes you can use touch() from there. Actually there is createFiles(count) but that is in S3ATestUtils.